client/config/index.js

```diff
@@ -17,13 +17,82 @@ for (let chainId in contracts) {
   contracts[chainId] = tmp;
 }
 
-const supportedId = {};
-supportedId[56] = "BSC";
-supportedId[97] = "BSC Testnet";
-supportedId[1337] = "localhost";
+function toHex(val) {
+  return "0x" + Number(val).toString(16);
+}
+
+let isDev;
+let isTestnet;
+let isStage;
+let isProd;
+let isJero;
+
+if (typeof window !== "undefined") {
+  const { hostname } = window.location;
+  isDev = /localhost/.test(hostname);
+  isStage = /(staking-stage|jeroyafra).mob.land(|\.local)$/.test(hostname);
```

Comment:
We do not have this for blueprints. From that configuration we should just get `isDev` and use it to set something as supported or not. Also, the format of supported can just be
```
supportedId[97] = isDev ? "BSC Testnet" : undefined;
```
if we do not need the information to manage the switch between chains (which is why in staking we have a different format).
In general, let's use only what is necessary, if not a future reader will ask why we put that things if is never used.

Reply:
BTW, the info we added now in `supportedId` are in the following field `chainConf`